content/2015.12.what-can-kinto-do.rst

```diff
@@ -0,0 +1,116 @@
+What can Kinto do for me?
+#########################
+
+:date: 2015-12-26
+:slug: what-can-kinto-do-for-you
+:lang: en
+:url: what-can-kinto-do-for-you
+:summary: Kinto, the project we're working on can do a bunch of different
+          things, bit it's sometimes hard to see what it's really good at, and
+          where it would make sense to use other solutions.
+
+Kinto — the project we're working on since quite some time now — can do a bunch
+of different things for you and be useful in many different situations.
+
+However, we found out that it's really hard to understand at a glance what it
+can do, and where it could help you build your projects in a better way.
+
+Here's an attempt to fix this.
+
+Kinto is not a framework
+########################
+
+First and foremost, Kinto is not a framework. Kinto is meant to replace the
+storage layer of your application, or more accurately, the synchronisation
+abilities of your applications.
+
+Said differentely, you store your data locally for your application and Kinto
+will make this data available to different devices. That means that you can use
+Kinto with React, Angular, JQuery (in the JavaScript world) or any framework in
+any language. Kinto abstracts the synchronisation of the application's data.
+
+Some examples
+#############
+
+Let's take some examples. A few weeks ago we prepared a bunch of demos
+showcasing what Kinto can do and how easy it is. In order to do that, we
+decided to take a few applications we liked that are usually not doing
+synchrnisation sharing. There are many out there, and we chose just a few of
+them:
+
+- **A JQuery calendar application** which let you add new events and see them
+  on a calendar.
+- **A leaflet map** where you can add new markers and move them easily.
+- **A TODO-list application**, because we actually really like TODO lists.
+
+And then we decided to make them collaborative apps, using Kinto. The goal here
+was to spend the least time possible plugging Kinto, in order to see what was
+the real potential of it.
+
+We ended-up with a few cool demos. Not all of them are using the same features
+of Kinto (for instance some take advantage of the push notifications while some
+others don't), but overall that was pretty exciting.
+
+Here are some links if you want to try out the demos yourself:
+
+* `Kinto.js + Leaflet <http://leplatrem.github.io/kinto-demo-leaflet/>`_ which
+  requires service workers activated (only works in Firefox Nightly these
+  days).
+* `Kinto.js + fullCalendar <http://leplatrem.github.io/kinto-demo-calendar/>`_
+* `Kinto.js TodoMVC <http://natim.github.io/kinto-demo-todomvc>`_
```

Comment:
We should add the cliquet-pusher demo as well as the Kinto-Telegram-Wall.

[0] http://leplatrem.github.io/cliquet-pusher/
[1] http://leplatrem.github.io/kinto-telegram-wall/